### DISCUSSION TITLE
q-page-scroller not functioning inside q-scroll-area

### DISCUSSION THREAD

**Discussion by Mbensler** [quasarframework/quasar#13112] [General - Components / Directives / etc]
Hello,
It seems like `q-page-scroller` is not functioning inside a `q-scroll-area` component. I can get it work outside of a `q-scroll-area` but not inside. Is there a way to get it to work inside a `q-scroll-area`?

```js
// BELOW DOESN'T WORK! BUT IF YOU DELETE THE q-scroll-area component IT WORKS!
    <q-scroll-area class="window-height window-width">
      <div v-for="n in 100" :key="n" class="q-py-xs">
        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
        tempor incididunt ut labore et dolore magna aliqua.
      </div>
      <q-page-scroller
        position="bottom-right"
        :scroll-offset="150"
        :offset="[180, 108]"
      >
        <q-btn fab icon="keyboard_arrow_up" color="accent" />
      </q-page-scroller>
    </q-scroll-area>
```

**Comment by zorenkonte** [comment]
Hello, did you figure it out? I'm having the same issue.

without q-scroll-area
codepen: https://codepen.io/zorenkonte/pen/dygzKOx 

with q-scroll-area
codepen: https://codepen.io/zorenkonte/pen/OJBjEXX

**Reply by jomiklos** [reply]
Did any of you manage to get it working with keyboard?
I observe same behavior with latest quasar - i cannot get keyboard to work with q-scroll-area (inside of  q-dialog) 
- trying to manually invoke `.focus()` on HMLT reference to `q-scroll-area` does not help (focus == undefined) 
- adding `tabindex` seems not to change anything (focus == undefined still)
